cms/src/symmetric_key.rs

```diff
@@ -0,0 +1,238 @@
+//! SymmetricKeyPackage types
+
+use alloc::vec::Vec;
+use der::Enumerated;
+use der::Sequence;
+use der::asn1::{OctetString, Utf8StringRef};
+use x509_cert::attr::Attribute;
+
+/// The `SymmetricKeyPackage` type is defined in [RFC 6031 Section 2.0].
+///
+/// ```text
+///      SymmetricKeyPackage ::= SEQUENCE {
+///        version           KeyPkgVersion DEFAULT v1,
+///        sKeyPkgAttrs  [0] SEQUENCE SIZE (1..MAX) OF Attribute
+///                                       {{ SKeyPkgAttributes }} OPTIONAL,
+///        sKeys             SymmetricKeys,
+///        ... }
+/// ```
+///
+/// [RFC 6031 Section 2.0]: https://datatracker.ietf.org/doc/html/rfc6031#section-2
+#[derive(Sequence, PartialEq, Eq)]
+#[allow(missing_docs)]
+pub struct SymmetricKeyPackage {
+    #[asn1(default = "Default::default")]
+    pub version: KeyPkgVersion,
+    #[asn1(context_specific = "0", tag_mode = "IMPLICIT", optional = "true")]
+    pub s_key_pkg_attrs: Option<Vec<Attribute>>,
+    pub skeys: SymmetricKeys,
```

Comment:
```suggestion
    pub s_keys: SymmetricKeys,
```

Just for consistency with the `sKeyPkgAttrs` field